### ISSUE TITLE
Way to indicate invalid code

### ISSUE THREAD

**Issue by MelSumner** [carbon-app/carbon#1014]
I love using carbon when I need to include code samples in my slides. I'm often talking about incorrect code, though, and how to fix it- so I'd love a way to indicate that a line of code should be marked incorrect. 

By adding a toggle that applied something similar to the following CSS, to a line of code, it could perhaps visually imitate other IDEs: 

```css
text-decoration: red wavy underline;
text-decoration-skip-ink: none;
```

Here's an image: 
![carbon (10)](https://user-images.githubusercontent.com/4587451/82343987-1d6e7880-99b9-11ea-8a73-b8fc1fd342b3.png)

Thank you for considering!

**Comment by mfix22** [comment]
This is a fantastic idea! I was so excited about it tonight, I implemented the initial version (here https://github.com/carbon-app/carbon/pull/1015)

Once released, if you select some text, and click the _"U"_ twice, the text will get underlined as errant, as shown here:
![Kapture 2020-05-19 at 21 04 31](https://user-images.githubusercontent.com/8397708/82403779-859e7800-9a14-11ea-8933-5af09a06d4c8.gif)

It's not perfect (squiggles between `span`s sometimes render a gap), so we don't have to mark this issue closed quite yet, but I wanted to unblock your use case at least a little bit 🙂

**Comment by mfix22** [comment]
@all-contributors add @MelSumner for ideas

**Comment by MelSumner** [comment]
Oh this is amazing, thank you so much!! 💯

**Comment by mfix22** [comment]
Gonna close this as complete! Feel free to open a new issue if enhancements are desired.
